nomic/tests/test_atlas_client.py

```diff
@@ -188,6 +188,31 @@ def test_topics():
 
         project.delete()
 
+def test_data():
+    num_embeddings = 100
+    embeddings = np.random.rand(num_embeddings, 10)
+    texts = ['foo', 'bar', 'baz', 'bat']
+    data = [
+        {'field': str(uuid.uuid4()), 'id': str(uuid.uuid4()), 'upload': 0.0, 'text': texts[i % 4]}
```

Comment:
Could we change this to `'text': str(i)` for this test? Once we remove `colorable_fields`, this might get auto-inferred as a categorical field which would mean this test would no longer be testing what it's supposed to.